tidb-cloud/sql-proxy-account.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SQL Proxy Account
+summary: Learn about the SQL proxy account in TiDB Cloud.
+---
+
+# SQL Proxy Account
+
+A SQL proxy account is a SQL user account that is automatically created by TiDB Cloud to access the database by SQL Editor and Data Service on behalf of a TiDB Cloud user. For example, `testuser@pingcap.com` is a TiDB Cloud user account, while `3jhEcSimm7keKP8.testuser._41mqK6H4` is its corresponding SQL proxy account.
+
+SQL proxy accounts are needed for the SQL Editor or Data Service to access the database on behalf of a TiDB Cloud user.
+
+## Identify the SQL proxy account
+
+If you want to identify whether a specific SQL account is a SQL proxy account, take the following steps:
+
+1. Examine the `mysql.user` table:
+
+    ```sql
+    USE mysql;
+    SELECT user FROM user WHERE plugin = 'tidb_auth_token';
+    ```
+
+2. Check grants for the SQL account. If roles like `role_admin`, `role_readonly`, or `role_readwrite` are listed, then it is a SQL proxy account.
+
+    ```sql
+    SHOW GRANTS for 'username';
+    ```
+
+## SQL proxy account creation
```

Comment:
Perhaps rename to "how SQL Proxy Accounts are Created"

Reply:
Also perhaps grouping account creation and deletion into a section "Managing SQL Proxy Accounts"

Reply:
> Also perhaps grouping account creation and deletion into a section "Managing SQL Proxy Accounts"

Since creating the SQL proxy account is not user action, using verbs like "managing" can be confusing.